Tutorials/ASP.NET/Blazor.Client/README.MD

```diff
@@ -0,0 +1,89 @@
+## Overview
+
+This tutorial demonstrates how to create a client-side Blazor application that uses the DevExpress ORM Library (XPO) to access and manage data. In this tutorial, we use an [intermediate Web API service](../../WebApi) to communicate with the database.
+
+<p align="center">
+  <img width="562" src="/Tutorials/images/Blazor.ClientSide/1.1.png">
+</p>
+<p align="center">
+  <img width="800" src="/Tutorials/images/Blazor.ClientSide/1.2.png">
+</p>
+
+## Prerequisites
+
+* [Visual Studio 2019 16.4.5 or later](https://visualstudio.com/) with the following workloads:
+  * **ASP.NET and web development**
+  * **.NET Core cross-platform development**
+* [.NET Core 5.0 SDK or later](https://www.microsoft.com/net/download/all)
+* [DevExpress UI Components for Blazor](https://www.devexpress.com/blazor/)
+
+This app uses the [WebAssembly hosting model](https://docs.microsoft.com/en-us/aspnet/core/blazor/hosting-models?view=aspnetcore-5.0#blazor-webassembly).
+
+## Step 1: Create a Blazor (Server-Side) Web App
+
+1. Open Visual Studio and create a new project.
+2. Find and select the **Blazor App** template. Click **Next** to proceed.
+3. Set the project name and click **Create**.
+4. Select **.NET 5.0** and **Blazor WebAssembly App** and click **Create**.
+
+## Step 2: Add the XPO Package
+
+Use any of the following approaches to install the XPO package:
+
+**Package Manager Console**
+
+  * From the Visual Studio **Tools** menu, select **NuGet Package Manager > Package Manager Console**.
+  * Make sure **Package source** is set to **All**, **nuget.org** or **DevExpress 20.2 Local** and run the following command: 
+     ```console
+    Install-Package DevExpress.Xpo -Version 20.2
+    ```
+
+**NuGet Package Manager**
+
+  * In **Solution Explorer**, right-click the project name and select **Manage NuGet Packages**.
+  * Make sure **Package source** is set to **All**, **nuget.org** or **DevExpress 20.2 Local** on the right.
+  * In the top left corner under **Browse**, search for the `DevExpress.Xpo` package and select the latest version, v20.2, on the right.
+  * Click **Install** and accept all licenses in the **License Acceptance** dialogs. 
+
+For more information, see the following:
+
+  * https://www.nuget.org/
+  * https://nuget.devexpress.com/
+
+## Step 3: Create ORM Data Model
+
+Refer to the following help topic to learn how to accomplish this task: [Create a Data Model](https://docs.devexpress.com/XPO/401951/create-a-data-model).
```

Comment:
- "to learn how to" You do not learn something - a topic contains instructions or information
- "to accomplish this task" Which task?

Reply:
Or you can say:  
Refer to the following help for instructions how to accomplish this task: [Create a Data Model](https://docs.devexpress.com/XPO/401951/create-a-data-model).

Refer to the following help topic for instructions on how to use XPO to create a database: [Create a Data Model](https://docs.devexpress.com/XPO/401951/create-a-data-model).